src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -58,14 +58,18 @@ public IActionResult Status()
             {
                 Version = this.fullNode.Version?.ToString() ?? "0",
                 Agent = this.nodeSettings.Agent,
-                PID = Process.GetCurrentProcess().Id,
+                ProcessId = Process.GetCurrentProcess().Id,
                 Network = this.fullNode.Network.Name,
                 ConsensusHeight = this.chainState.ConsensusTip.Height,
-                BlockStoreHeight = storeStateProvider.StoreTip.Height,
                 DataDirectoryPath = this.nodeSettings.DataDir,
                 RunningTime = this.dateTimeProvider.GetUtcNow() - this.fullNode.StartTime
             };
 
+            // Include BlockStore Height if enabled
+            foreach (var blockStore in this.fullNode.Services.Features)
```

Comment:
that iteration variable is not a block store it is a feature that might not be block store, so it is kind of misleading to call it block store